package.json

```diff
@@ -89,6 +90,8 @@
   "jsonwebtoken": "^8.5.1",
   "multer": "^1.4.2",
   "pg": "^8.3.0",
+  "pino": "^6.5.1",
+  "pino-pretty": "^4.2.0",
```

Comment:
Can you install this as a dev dependency instead? It's just because the application doesn't _need_ pino-pretty to actually run.